can-stream-kefir.js

```diff
@@ -85,4 +85,4 @@ canStreamKefir.toCompute = function(makeStream, context){
 /*
  * Exposes a simple toStream method that takes an observable and event or propname and returns a Kefir stream object
  */
-module.exports = canStream(canStreamKefir);
+module.exports = namespace.canStreamKefir = canStream(canStreamKefir);
```

Comment:
Sorry to nit-pick, but we'd like it to be `can.streamKefir`. This would make it `can.canStreamKefir` which is a little redundant.

Reply:
Not a problem. That's a good point. I've updated the PR.